### ISSUE TITLE
How should missing values in the output be treated?

### ISSUE THREAD

**Issue by brendanwee** [tyjo/coptr#11]
In the final output of my experiment, I get a decently populated matrix. I wish to incorporate this matrix into a machine learning model. Looking through the paper, I didn't see any mention of having to handle missing values. (unless I missed it). How would you treat missing values? Assuming that they cannot be ignored

**Comment by tyjo** [comment]
That's a good question. The analyses in the paper only compared non-missing PTRs within the same species.

Missing values are challenging to deal with because they could be missing for two reasons:
1) The species is not in the sample, or
2) The species is below the detection threshold.

If you could distinguish the two types of missing values, you could try to impute missing values of the type 2. However, it's not clear to me how to set missing values for species that are not in a sample.

If your model starts by taking a linear combination of an input vector, one possibility is to set missing values to 0. I am not sure how well this will work in practice.

**Comment by brendanwee** [comment]
hmm... If you reduced the threshold parameters on the CoPTR all the way, ex. `min_reads`, `mincov`,  would it predict a PTR for all species with a read mapped? In that case you be left with only #1 ?

**Comment by tyjo** [comment]
Even if you lowered the threshold, you likely won't get a full matrix. There could be low abundance species in a sample without any sequencing reads.

**Comment by brendanwee** [comment]
yeah... Thanks for the ideas. I'll let you know if we come up with something useful!
